### PR TITLE
fix: replace imgur URLs with s3 URLs for files with potential conflict

### PR DESCRIPTION
**Pull request by ahmaxed**
This pr is a collection of all of #36007 files that might conflict with other pull requests based on the contributor tool.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Reference #36007

### CODE REVIEW

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb The other PR has been merged now and it appears there are no merge conflicts.  Yeah!!!

**Comment by ahmadabdolsaheb** [comment]
@RandellDawson thanks for taking the time to go over these, you could approve this and see if there are merge conflicts. you can always go back.

File: guide/russian/cplusplus/index.md
Comment (RandellDawson) [inline guide/russian/cplusplus/index.md]: Is this line needed?
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ahmadabdolsaheb I only have one suggested change.

File: guide/russian/cplusplus/index.md
Comment (ahmadabdolsaheb) [reply guide/russian/cplusplus/index.md]: @RandellDawson how about replacing 
`Img`
with
`Img`
instead of removing
`https://cdn-media-1.freecodecamp.org/imgr/`

**Comment by ahmadabdolsaheb** [comment]
@RandellDawson, thanks.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb OK, this PR looks good now.

**Comment by ahmadabdolsaheb** [comment]
I double checked this. it should be good to go.

**Comment by RandellDawson** [comment]
@thecodingaviator Are you good with these changes?

**Review by thecodingaviator — APPROVED** [review]
Changes LGTM. I haven't gone through the files to check for any missed conversions

**Comment by RandellDawson** [comment]
@thecodingaviator @ahmadabdolsaheb There are currently 22 or 23 earlier PRs with one or more files.  Do we wait to merge this PR until those have been reviewed/merged or can we go ahead and merge this PR now?

**Comment by thecodingaviator** [comment]
I think we can merge this since I don’t think any PR would be making changes to any image

**Comment by ahmadabdolsaheb** [comment]
@RandellDawson @thecodingaviator, sorry for the late replies these days.
we could merge this and check affected prs to see if they get conflicts. 
if they do, we can always revert this one.  would that work?

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb I am fine with that.  If it is only a few PRs with merge conflicts that are English, then I can easily resolve them.  If there a lot of non-English PRs, then I would rather revert.  I have the list of PRs which could be affected and will check right after we merge.

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb There were about 5 PRs with merge conflicts which I was able to resolve.  There was one PR (https://github.com/freeCodeCamp/freeCodeCamp/pull/33593) which now has a merge conflict, but it is a Russian translation and has a massive merge conflict now.  I am going to put a note on it to let the reviewer know to make sure to keep the cdn image and not the imgur image when resolving the merge conflict.

**Comment by ahmadabdolsaheb** [comment]
Thank you @RandellDawson, I really appreciate your efforts.

**Comment by RandellDawson** [comment]
Isn't there still another open PR like this one that could have merge conflicts?

**Pull request merged**
